packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -294,7 +300,21 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
         },
         include: {
           attendees: true,
-          eventType: true,
+          eventType: {
+            select: {
+              slug: true,
+              schedulingType: true,
+              hosts: {
+                select: {
+                  user: {
+                    select: {
```

Comment:
Tried using variable, typescript compiler wasn't having it so had to repeat this